metadata.rb

```diff
@@ -4,7 +4,7 @@
 license 'Apache 2.0'
 description 'Installs activemq and sets it up as service'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version '2.0.2'
+version '2.0.3'
```

Comment:
That's fine with me, I'll change it.
On Oct 8, 2015 3:09 AM, "Barthélemy Vessemont" <notifications@github.com>
wrote:

> In metadata.rb
> <https://github.com/chef-cookbooks/activemq/pull/30#discussion_r41496618>:
>
> > @@ -4,7 +4,7 @@
> >  license 'Apache 2.0'
> >  description 'Installs activemq and sets it up as service'
> >  long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
> > -version '2.0.2'
> > +version '2.0.3'
>
> IMO PR should *not* bump cookbook version.
> What's the best practice here ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/chef-cookbooks/activemq/pull/30/files#r41496618>.
>